### PR TITLE
Fix Attack Command Number; Add Chorus & Portamento Time

### PR DESCRIPTION
**Pull request by deckerego**
- Fixed ATTACK_TIME value
- Added CHORUS value
- Added PORTAMENTO_TIME to match the PORTAMENTO on/off value

### CODE REVIEW

File: adafruit_midi/control_change_values.py [hunk 1]
```diff
@@ -46,13 +46,15 @@
 EXPRESSION = 11
 SUSTAIN_PEDAL = 64
 PORTAMENTO = 65
+PORTAMENTO_TIME = 37
```
Comment (tekktrik) [inline adafruit_midi/control_change_values.py:49]: Should this be `5`?

Comment (tekktrik) [reply adafruit_midi/control_change_values.py:49]: Also, would you mind moving it, whatever the value is/should be so that it's ordered by value?  Thanks!

**Review by tekktrik — COMMENTED** [review]
I know very little about MIDI so I don't know if I'm qualified approve this or not, but a quick search about it lead to the above comments.  Thanks for submitting!

File: adafruit_midi/control_change_values.py (hunk 1 above)
Comment (deckerego) [reply adafruit_midi/control_change_values.py:49]: `5` is for the coarse adjustment value, `37` is the fine adjustment value. Will adjust & order by numeric value.

Comment (tekktrik) [reply adafruit_midi/control_change_values.py]: Sounds good!  I'll mark this as resolved and wait for someone more knowledgeable make the call :D

**Comment by rimwolf-redux** [comment]
The putative norms and actual practice of which CCs should be used for which controls diverges widely, and this file by and large defines just the most commonly implemented assignments; many synthesizers won't even implement all of them thus. (It might be wise to add a comment to the file to this effect.) My comments address these common usages.

PORTAMENTO_TIME should be 5, not 37. Theoretically CCs 32--63 are reserved for LSB (7 bits) of the MSB (7 bits) of the controller values for 0--32, but in practice very few synthesizers implement LSB for any of these values. (CC37 would be the LSB of CC5.) I certainly don't think it's worthwhile defining names for any of them.

Another relatively common controller is BANK SELECT, CC 0. This is used to select which bank of patches a Program Change message references.

The assigments for CCs 75--78 are not common -- they're normatively just Sound Controller 6--9 -- and I'd be happy to see them removed from this file.

(These comments are based on my having studied, owned, and used a wide variety of synthesizers, hardware and software, since the 1980s, with particular attention to CC implementation.)

**Comment by deckerego** [comment]
> ...it looks like `5` is the MSB for the coarse adjustment, and `37` is the LSB for fine adjustment. In that case, it might be more future proof to label `37` as `PORTAMENTO_TIME_FINE` so that `PORTAMENTO_TIME` can be `5`.

Updated to coarse adjustment

**Pull request merged**